tests/test_paperqa.py

```diff
@@ -683,7 +683,7 @@ def test_sparse_embedding(stub_data_dir: Path, vector_store: type[VectorStore])
         citation="WikiMedia Foundation, 2023, Accessed now",
         embedding_model=SparseEmbeddingModel(),
     )
-    assert any(docs.texts[0].embedding)
```

Comment:
Can we keep the `any` assertions around? Why move to `list`?

If you think the assertion is the same behavior, can you add an assertion failure message, to help make what it's checking for clear?

```diff
--- assert any(docs.texts[0].embedding), "We require the embedding to be populated"
+++ assert isinstance(docs.texts[0].embedding, list), "We require the embedding to be populated"
```